sources/forms-docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+services:
```

Comment:
Why do we need to put forms in separate docker compose file?

Reply:
formio server is a separate server and can be started separately like camunda, so created separately.

Reply:
I see the point from @dheepak-aot and I am ok to proceed as it is right now but we can talk about it.
I am having second thoughts even for Camunda. Maybe we can revisit it and have it all together.
Right now adding all to the same will delay the API test running on Github. I guess that it will build all the docker-compose and it will add unnecessary overhead, unless I am wrong 😄